components/dashboard/DashboardBlocks.js

```diff
@@ -43,69 +45,30 @@ const DashboardBlocks = () => (
         </div>
       </Col>
       <Col>
-        <Row className="gutter-4">
-          <DashboardBlocksBlock
-            blockNumber="6066099"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066098"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066099"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066098"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066099"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066098"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066099"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <DashboardBlocksBlock
-            blockNumber="6066098"
-            blockHash="1a2b3c4d5e6f"
-            transactionsCount="254"
-            blockHeight="15489"
-            witness="vlad.near"
-          />
-          <Col xs="6">
-            <Link href="blocks">
-              <a className="dashboard-footer">View All</a>
-            </Link>
-          </Col>
-        </Row>
+        <DataConsumer>
+          {context => (
+            <Row className="gutter-4">
+              {context.blocks.map((block, index) => {
+                return (
+                  <DashboardBlocksBlock
+                    key={context.blocks[index].blockNumber}
```

Comment:
Can we use `block` instead of `context.block[index]`?

Reply:
Yes. We can. My mistake. Will fix it now.

Reply:
Updated